app/src/main/java/com/safetyheads/akademiaandroida/Footer.kt

```diff
@@ -0,0 +1,86 @@
+package com.safetyheads.akademiaandroida
+
+import android.content.Intent
+import android.graphics.Color
+import android.net.Uri
+import android.os.Bundle
+import android.text.SpannableString
+import android.text.Spanned
+import android.text.TextPaint
+import android.text.method.LinkMovementMethod
+import android.text.style.ClickableSpan
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+
+// Class responsible for displaying a footer at the bottom of the screen with clickable links
+// to the "Privacy policy" and "Cookie policy" pages.
+class Footer : Fragment() {
+    // Utility function for creating clickable spans
+    private fun createClickableSpan(spannableString: SpannableString, start: Int, end: Int, url: String) {
+        spannableString.setSpan(object : ClickableSpan() {
+            override fun onClick(widget: View) {
+                val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
+                requireContext().startActivity(browserIntent)
+            }
+
+            override fun updateDrawState(ds: TextPaint) {
+                super.updateDrawState(ds)
+//Todo: Replace hardcoded values
+                ds.color = Color.BLUE
+                ds.isUnderlineText = false
+            }
+        }, start, end, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE)
+
+
+    }
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        val view = inflater.inflate(R.layout.fragment_footer, container, false)
```

Comment:
Rozwiązanie na pewno działa, aczkolwiek nie możemy zmergować tego w takiej postaci, trzeba trochę to upiększyć. Masz w tym momencie dwa rozwiązania, jedno prostsze i jedno nieco bardziej skomplikowane aczkolwiek wygodniejsze na przyszłość.
1. Cały ten sposób na kolorowanie i podpinanie linków trzeba zamknąć w metodzie xyz przyjmującej tekst + listę obiektów typu Pair składających się z Stringa oznaczającego jaki tekst chcemy pokolorować oraz jaki opcjonalnie link do niego podpiąć. Metoda powinna też przyjmować parametr na jaki kolor chcemy pokolorować tekst i na koniec zwrócić SpannableString. Warto stworzyć package utlis a w niej TextUtils.kt, w którym taka metoda by się znalazła.
2. Alternatywnie stworzyć custom view, który posiadałby powyższą funkcję.

W razie pytań służę pomocą :)